yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -772,36 +764,48 @@ module ietf-yang-revisions {
   // RFC Ed.: replace XXXX (inc above) with actual RFC number and
   // remove this note.
 
-  revision 2022-10-10 {
+  revision 2022-11-21 {
     rev:revision-label 1.0.0-draft-ietf-netmod-yang-module-versioning-07;
```

Comment:
Change to -08?

Reply:
I suggest upping the revision after all outstanding action items are
resolved (or at when outstanding pull requests are merged).

Reply:
Why wait? The next rev is 08. Other AIs may not necessarily result in changes to this YANG module.

Reply:
I changed to -08.